docs/3.0.0-beta.x/concepts/customization.md

```diff
@@ -0,0 +1,31 @@
+# Customization
+
+## Plugin extensions
+
+In strapi you can install plugins in your `node_modules`. This allows for easy updates and respect best practices. To customize those installed plugins you can work in the `/extensions` directory. It contains all the plugins' customizable files.
+
+Certain plugins will create files in these folders so you can then modify them. You can also create certain files manually to add some custom configuration for example.
+
+Depending on the plugins you will find extension documentation directly in the plugin's documentation.
```

Comment:
Depending on the plugin.
I am not sure to understand the meaning of this sentence.